src/tools/mesh_tester.cpp

```diff
@@ -170,6 +181,9 @@ void Input::parse(int argc, char** argv, axom::CLI::App& app)
   #ifdef AXOM_USE_CUDA
   pol_sstr << "\nSet to \'raja_cuda\' or 3 to use the RAJA CUDA policy.";
   #endif
+  #if defined(AXOM_USE_HIP) && defined(NDEBUG)
+  pol_sstr << "\nSet to \'raja_hip\' or 4 to use the RAJA HIP policy.";
```

Comment:
Are the slashes necessary to escape single quotes within double quote strings? I think we can remove them.